djangocms_history/static/djangocms_history/color_mode.css

```
@@ -1,32 +1,43 @@
+html[data-theme="dark"] .history-button.redo  {
+    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAM5JREFUWEftlkEOwyAMBJPX8iZe24gDUkTt7Nikogdyxt5hvTI5j8XfuVj/2AAhB2qtHzKyUgruiw5S4RGug9zrRzgJkBX3nAoBvC3eoDDAL8QxgBL3QqbqpgFIuhWEHIHX4A1x5MBfApDbt9sp+9MOUACyJcczX4vIusUG2A5Y6c0EzqpBIcwA9DCr8ZnP8cwysvbBE0QIoFuYeYy8GveHhGy1SA7CAHS1EojwCO5NZ51IhXC8VRZCiTcd+VOacYMI974hADLv6JkNsNyBC7XYhCHbOi9lAAAAAElFTkSuQmCC');
 }
-html[data-theme="light"] .redo  {
-    content: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAABP0lEQVR42u3WP0vDQBiAcSXoYMWtpcYPoFjd3R01rn4Alx40Q9vv0q5+AxGsbiKOuoharM7OQv8MVqzng3Q6Ei5vLmQxD/zGcC9ckruFoiJhPhSuMMBk7g0XUNhA5q2jiy9oiyk6xiBl3OMG4gKMoIU+cIAKnqABWSFm0CnN8A4NQNBRzOJ9NFFDCSvYRB0PtqEkez6M2NsGPMS1CIVP1wG6EYvvI0llDFwG8CPe9oZg8UfXLVARe+4hSXdZvAOXxgNNJO3asvgtrL0aD20j18bGAKv/bgBzC2rILflLKM/9M3RP/iMKIW0Ly7Am/xXbCzDFedohqjGHUWjZDg8t4zA6wxLEBYg7jlvYQQlr2EUbz9CGH+xhXv4XkhM4dYhhyivZMTKpKriUfuMUFWSejzp6eMEYI/TRg/pbuKhI0C8BZPKBEtpCEgAAAABJRU5ErkJggg==');
+html[data-theme="light"] .history-button.redo  {
+    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAABP0lEQVR42u3WP0vDQBiAcSXoYMWtpcYPoFjd3R01rn4Alx40Q9vv0q5+AxGsbiKOuoharM7OQv8MVqzng3Q6Ei5vLmQxD/zGcC9ckruFoiJhPhSuMMBk7g0XUNhA5q2jiy9oiyk6xiBl3OMG4gKMoIU+cIAKnqABWSFm0CnN8A4NQNBRzOJ9NFFDCSvYRB0PtqEkez6M2NsGPMS1CIVP1wG6EYvvI0llDFwG8CPe9oZg8UfXLVARe+4hSXdZvAOXxgNNJO3asvgtrL0aD20j18bGAKv/bgBzC2rILflLKM/9M3RP/iMKIW0Ly7Am/xXbCzDFedohqjGHUWjZDg8t4zA6wxLEBYg7jlvYQQlr2EUbz9CGH+xhXv4XkhM4dYhhyivZMTKpKriUfuMUFWSejzp6eMEYI/TRg/pbuKhI0C8BZPKBEtpCEgAAAABJRU5ErkJggg==');
 }
 
 @media (prefers-color-scheme: light) {
-    html[data-theme="auto"] .undo, html:not([data-theme]) .undo {
-        content: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAM9JREFUWEftlksOwCAIRPW0nsnTtnFBYqwwg/GTJnYrn+dAwRgOf/Fw/nABXArknB+mZCklOu7HsE4igdjELRwDYgIwt0U2CGI5QAG0ILYAWBBDANqNUK/0/NwAqKblthrIFABUU2lKFsKtwG8AtFK0ZaAnFvrfe+e9MlyAq8A2BYbnQNvNEoiZgLXvFIA2CAvBJi/A6hywFsvSZYRm+chAcq1jpo5eCNeDRGtCb1KxR31D7QL00NDgUHKzCdnl0rNjEosfpcCo/IzfBTiuwAuFfoQhCFHtvgAAAABJRU5ErkJggg==');
+    html[data-theme="auto"] .history-button.undo, html:not([data-theme]) .history-button.undo {
+        background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAM9JREFUWEftlksOwCAIRPW0nsnTtnFBYqwwg/GTJnYrn+dAwRgOf/Fw/nABXArknB+mZCklOu7HsE4igdjELRwDYgIwt0U2CGI5QAG0ILYAWBBDANqNUK/0/NwAqKblthrIFABUU2lKFsKtwG8AtFK0ZaAnFvrfe+e9MlyAq8A2BYbnQNvNEoiZgLXvFIA2CAvBJi/A6hywFsvSZYRm+chAcq1jpo5eCNeDRGtCb1KxR31D7QL00NDgUHKzCdnl0rNjEosfpcCo/IzfBTiuwAuFfoQhCFHtvgAAAABJRU5ErkJggg==');
```

Would this not be the light image identical to line 5?
```suggestion
        background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAABOklEQVR4Ae3WNVZDQRhAYdyhimcBeMsG6CI1C8Dd9pK0rACXCm+QDoeaFnfnFlNNns4bnHvO10T/55Plsv/+i6ITMzjCtXCAOfFeBJ5axCb80h+n8YA3G49IIwSl3oRtBBDHKd5cukBSeQDhGC94U/SCXvUBjO2gF9UoFWoxiF2TIZI6BrhHF3JgVq4Y7sHgcIS8DnCEAJzUaDBESsch2IYfTuo1uDoiOs6BdTgp1+Cc6IBtqzYDLMBpg9J3p/Gp1UoDHOBTK5MGuMKnViFfjn/uEAx95UmofBnKFaAKbuszuBGFVf58Ag9IgLzfip2WjzHpYTSIXIWH0TmCcFUDXk0ex8OoRwXKUIchi8dxAkq1fMWCRK5JcUl2jgS0FMAInh0uSlPyMdc5SBdmsItLXGEfM+hwe6n99987pVTygpWk5ykAAAAASUVORK5CYII=');
```